pkg/proc/breakpoints.go

```diff
@@ -4,17 +4,16 @@ import (
 	"debug/dwarf"
 	"errors"
 	"fmt"
-	"go/ast"
-	"go/constant"
-	"go/parser"
-	"go/token"
-	"reflect"
-
 	"github.com/go-delve/delve/pkg/dwarf/godwarf"
 	"github.com/go-delve/delve/pkg/dwarf/op"
 	"github.com/go-delve/delve/pkg/dwarf/reader"
 	"github.com/go-delve/delve/pkg/goversion"
 	"github.com/go-delve/delve/pkg/proc/internal/ebpf"
+	"go/ast"
```

Comment:
I'm pretty sure the standard convention is to keep standard library packages in one block and other imports in another.

Reply:
For some reason, the linter I was using did that weird thing. Fixed.